test/routes/review.test.js

```diff
@@ -286,6 +349,68 @@ describe('/review/', () => {
           expect(res.text).toEqual(expect.not.arrayContaining(['id="withdrawAddress"']))
         })
     })
+    test('Shows version 1 of the occupiers consent to HDC text', () => {
+      licenceService.getLicence = jest.fn().mockReturnValue({
+        licence: {
+          eligibility: {
+            proposedAddress: {
+              addressLine1: 'line1',
+            },
+          },
+        },
+        stage: 'APPROVAL',
+      })
+      conditionsService.populateLicenceWithConditions = jest.fn().mockReturnValue({
+        curfew: {
+          curfewAddressReview: {
+            version: '1',
+            addressReviewComments: '',
+            consent: true,
+          },
+        },
+        proposedAddress: { curfewAddress: { addressLine1: 'line1' } },
+      })
+
+      return request(app)
+        .get('/hdc/review/address/1')
+        .expect(200)
+        .expect('Content-Type', /html/)
+        .expect((res) => {
+          expect(res.text).toContain('Does the main occupier consent to HDC?')
+        })
+    })
+
+    test('Shows version 2 of the occupiers consent to HDC text', () => {
+      licenceService.getLicence = jest.fn().mockReturnValue({
+        licence: {
+          eligibility: {
+            proposedAddress: {
+              addressLine1: 'line1',
+            },
+          },
+        },
+        stage: 'APPROVAL',
+      })
+
+      conditionsService.populateLicenceWithConditions = jest.fn().mockReturnValue({
+        curfew: {
+          curfewAddressReview: {
+            version: '2',
+            addressReviewComments: '',
+            consent: true,
+          },
+        },
+        proposedAddress: { curfewAddress: { addressLine1: 'line1' } },
+      })
+
+      return request(app)
+        .get('/hdc/review/address/1')
+        .expect(200)
+        .expect('Content-Type', /html/)
+        .expect((res) => {
+          expect(res.text).toContain('Have you spoken to the main occupier and do they consent to HDC?')
+        })
+    })
```

Comment:
Do we need all added tests, might be getting confused but I feel like there is some duplication - though my brain might just be struggling 😅

Reply:
The reason for what appear to be the same tests is that one set is for the PROCESSING_RO stage and the other for the APPROVAL stage.